google/cloud/bigquery/client.py

```diff
@@ -1549,10 +1723,20 @@ def cancel_job(
             extra_params["location"] = location
 
         path = "/projects/{}/jobs/{}/cancel".format(project, job_id)
+        span_creator = SpanCreator()
 
-        resource = self._call_api(
-            retry, method="POST", path=path, query_params=extra_params, timeout=timeout
-        )
+        attributes = {"path": path, "job_id": job_id, "location": location}
+
+        with span_creator.create(
+            name="BigQuery.CancelJob", attributes=attributes, client=self
```

Comment:
Most of the methods used lower case for the first letter; should this name be `BigQuery.cancelJob`?